middleware/jwt_auth.go

```diff
@@ -74,6 +74,6 @@ func verifyToken(header string, secret string) (*jwt.Token, *authorisationError)
 
 func hashEmail(email string, salt string) string {
 	hash := sha256.New()
-	hash.Write([]byte(salt + email))
+	_, _ = hash.Write([]byte(salt + email))
```

Comment:
Is this a Golang convention?

Reply:
Gosec throws an error without the `_, _` bit. I guess it forces you to explicitly ignore returned errors/values for better visibility